src/vite.rs

```diff
@@ -34,16 +34,16 @@ use serde::Deserialize;
 use sha1::{Digest, Sha1};
 use std::collections::HashMap;
 
-pub struct Development {
+pub struct Development<'a> {
```

Comment:
This is necessary because of the `layout` closure. In order to be thread-safe, the type of the closure's return needs to implement `Send + Sync + 'static`. By making it thread-safe, it's needed to say that 'a outlives 'static, but still it allows us to fill the struct `str` fields without having to leaking it or doing anything to make the string slice become `'static`.

Reply:
I'm still not sure I'm following. Why do we need a lifetime parameter at all? If I remove it and change the references to `'static`, there isn't a compilation issue.